editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5413,8 +5413,8 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	editor_selection->connect("selection_changed", this, "update");
 	editor_selection->connect("selection_changed", this, "_selection_changed");
 
-	editor->call_deferred("connect", "play_pressed", this, "_update_override_camera_button", make_binds(true));
-	editor->call_deferred("connect", "stop_pressed", this, "_update_override_camera_button", make_binds(false));
+	editor->call_deferred("connect", make_binds("play_pressed", this, "_update_override_camera_button", true));
+	editor->call_deferred("connect", make_binds("stop_pressed", this, "_update_override_camera_button", false));
```

Comment:
This does not seem correct to me. `make_binds creates` a `Vector<Variant>` out of the arguments passed to the function. However, there is no overload of `connect` function that would take a single parameter of type `Vector<Variant>`.

Reply:
I think MessageQueue (backend of call deferred) uses a Vector. Ideally we should fix varargs since they are poorly implemented :| that's something I need to work on in C++

Reply:
MessageQueue::push_call - the overload that Object::call_deferred calls - uses VARIANT_ARG for passing message parameters, not Vector<Variant>.